pre_commit_hooks/forbid_new_submodules.py

```diff
@@ -1,14 +1,32 @@
+import argparse
+import os
 from typing import Optional
 from typing import Sequence
 
 from pre_commit_hooks.util import cmd_output
 
 
 def main(argv: Optional[Sequence[str]] = None) -> int:
-    # `argv` is ignored, pre-commit will send us a list of files that we
-    # don't care about
+    parser = argparse.ArgumentParser()
+    parser.add_argument('filenames', nargs='*')
+    args = parser.parse_args(argv)
+
+    if not args.filenames:
+        return 0
```

Comment:
pre-commit will never call this with zero arguments

Reply:
Yeah, I've added it because of `git diff` or `git status` behavior difference with one or more path arguments (check them) vs. zero (check _all_, not none). But I agree that this is not needed.